### ISSUE TITLE
Getting error while adding convolution Layer conv1d_1 does not support masking

### ISSUE THREAD

**Issue by rajeshkumargp** [CyberZHG/keras-bert#104]
I am trying for CNN by Yoon Kim, with BERT encoding as features. 
Trying for the similar model https://github.com/inspirehep/magpie.

My intention is to build model
    keras-bert(without training) on pretrained model and 
    keras-bert.model's output is fed to Convolution for various grams. 

Code snippet is as below:

`import keras`
`import sys`
`import numpy as np`
`from keras_bert import load_vocabulary, load_trained_model_from_checkpoint, Tokenizer,` `get_checkpoint_paths`

     
`SEQ_LEN = 128`
`BATCH_SIZE = 128`
`EPOCHS = 5`
`LR = 1e-4`

`def get_nn_model():`
`  SEQ_LEN = SAMPLE_LENGTH = 128`
`  BATCH_SIZE = 128`
`  EPOCHS = 5`
`  LR = 1e-4`
`  output_length = 70`
  
`  pretrained_path = 'bert_model_base'`
`  config_path = os.path.join(pretrained_path, 'bert_config.json')`
`  checkpoint_path = os.path.join(pretrained_path, 'bert_model.ckpt')`
`  vocab_path = os.path.join(pretrained_path, 'vocab.txt')`
  
`  model = load_trained_model_from_checkpoint(`
`      config_path,`
`      checkpoint_path,`
`      training=False,`
`      trainable=False,`
`      seq_len=SEQ_LEN,`
`      )`
  
`  NB_FILTER = 256`
`  NGRAM_LENGTHS = [1, 2, 3, 4, 5]`
`  conv_layers, inputs = [], []`
`  for ngram_length in NGRAM_LENGTHS:`
`    #current_input = Input(shape=(SAMPLE_LENGTH, embedding_size))`
`    #inputs.append(current_input)`
    
`    convolution = keras.layers.Conv1D(`
`        filters=NB_FILTER,`
`        kernel_size=ngram_length,`
`        kernel_initializer='lecun_uniform',`
`        activation='tanh',`
`        )(model.output)`

`    pool_size = SAMPLE_LENGTH - ngram_length + 1`
`    pooling = MaxPooling1D(pool_size=pool_size)(convolution)`
`    conv_layers.append(pooling)`

`  merged = Concatenate()(conv_layers)`
`  print('merged',merged.shape)`
`  dropout = Dropout(0.5)(merged)`
`  print('dropout',dropout.shape)`
`  flattened = Flatten()(dropout)`
`  print('flattened',flattened.shape)`
  
`  outputs = Dense(output_length, activation='sigmoid')(flattened)`
`  print('outputs',outputs.shape)`
  
`  fin_model = Model(inputs=model.input, outputs=outputs)`
`  return fin_model`

`km = get_nn_model()`

While adding Convolution, I am getting error as if input is masked, and convolution cant be added. 
 
Error: 

`Layer conv1d_1 does not support masking, but was passed an input_mask: Tensor("Encoder-12-FeedForward-Add_1/All:0", shape=(?, 128), dtype=bool `
 

Can you please guide how to proceed further. .

Also, I made a try using MaskedConv1D(provided in keras-bert), but the next layers getting failed.
